static/app/components/searchQueryBuilder/useQueryBuilderGrid.tsx

```diff
@@ -17,59 +13,26 @@ interface UseQueryBuilderGridProps extends AriaGridListOptions<ParseResultToken>
 
 const noop = () => {};
 
-function findNearestFreeTextKey(
```

Comment:
Most of the changes in this file are moving the onKeyDown logic from there to the new component